ft-stripe/src/lib.rs

```diff
@@ -8,7 +8,7 @@ pub fn create_customer(
     let user_data = ft_sdk::auth::ud_from_session_key(conn, session_id)?
         .ok_or_else(|| CreateCustomerError::UserNotFound)?;
 
-    let body = match ft_sdk::auth::mobile_from_email(user_data.email.as_str()) {
+    let body = match mobile_from_email(user_data.email.as_str()) {
```

Comment:
No, this makes this stipe function not usable by virtually anyone else. Pass the body as a parameter to `create_customer()`. Checkout stripe's Rust SDK and mirror their `create_customer()` function call, we must do what that SDK does, there is a whole team in stripe deciding what is a good stripe API, no need to invent it here. We are creating this crate because stripe official sdk does not work.